plugins/com.google.cloud.tools.eclipse.sdk.test/pom.xml

```diff
@@ -1,10 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Interesting that here you're adding something that can be defaulted, and elsewhere you're taking it away. My personal preference is to include things that can be defaulted since I rarely remember what the defaults are. It also makes it easier to change to a non-default value when necessary.